### ISSUE TITLE
Contest mode. No duplicate checking?

### ISSUE THREAD

**Issue by m0urs** [magicbug/Cloudlog#1353]
**Describe the bug**

In the wiki it reads for the contest logging: "A basic duplicate check is implemented, which will tell you if the callsign is a duplicate on the same band, mode and contest."

**To Reproduce**

Steps to reproduce the behaviour:
1. Enter contest mode
2. Create an entry
3. Create an entry with same data callsign, band,mode and contest again
4. QSO is just saved in addition without alerting me that this is a duplicate

**Expected behaviour**

I would expect that there is an error popping up either after entering the call sign of a duplicate or at least while saving and a question is asked if you would like to save it even as a duplicate record.

**Comment by AndreasK79** [comment]
It's there:
![image](https://user-images.githubusercontent.com/6977712/148437370-8b5dc5c6-3e2f-44a2-b328-9d90eec8976e.png)
Nothing is stopping you from working/logging the dupes, it's just informing you that it is a dupe.
The policy should always be to work the dupes. It's less hassle to just work the dupes, than trying to explain that it is a dupe. You also never know if you really are in the log of the other station.

And if you are working S&P, then just don't log it.

I see the dupe check doesn't work for the first QSO. I will make a note of that, but for the following QSOs, it does work.

**Comment by m0urs** [comment]
Hm, unfortunately this message does not appear here :-( However, I am using the default theme. I just tried to change it to one of the dark themes but this does not work either.

I am using MS Edge (chrome based) and tried Firefox on Windows as well. Before I log in the start page of Cloudlog appears in the design I chose. After logging in, it changes to the default theme again.

In the developer console of Edge I can see this error message (although I am not sure if this is related):

`Fehler beim Laden der Quell Zuordnung durch devtools: Der Inhalt für https://my.server.com/assets/css/bootstrap-dialog.css.map konnte nicht geladen werden: HTTP Fehler: Statuscode 404,net::ERR_HTTP_RESPONSE_CODE_FAILURE`
